apps/pipeline/lib/pipeline/writer/table_writer/statement/rename.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Pipeline.Writer.TableWriter.Statement.Rename do
```

Comment:
Is it necessary to have this be separate from `Statement` module? Would it be better to make it a utility module instead, like `StatementUtils` or something like that?